src/api/features.api.js

```diff
@@ -0,0 +1,185 @@
+import { API_BASE_URL } from '@/config'
+import axios from 'axios'
+
+/**
+ * Describe a feature from the backend (see {@link getFeature}) below
+ */
+export class Feature {
+  /**
+   * @param {WMSLayer|WMTSLayer|GeoJsonLayer|AggregateLayer} layer the layer in which this feature belongs
```

Comment:
Tooltip for WMS, WMTS and GeoJSON is OK. For aggregated layer (e.g. _ch.swisstopo-vd.geometa-gemeinde_) is it the fall?

Reply:
Ho right, I have not thought about this use case.